code/modules/shuttle/special.dm

```diff
@@ -52,6 +52,9 @@
 /obj/machinery/power/emitter/energycannon/magical/emag_act(mob/user)
 	return
 
+/obj/machinery/power/emitter/energycannon/magical/RefreshParts()//monkestation edit, makes it correctly not require power
+	return ..()
```

Comment:
Returning parent just means that it's doing exactly what it already was.
Have you tested and confirmed that this change actually fixes the problem by calling this shuttle and checking?

Reply:
lint fails if I dont call parent. I did not originally call parent and it worked, but I will see if it works this way